src/vs/workbench/contrib/chat/browser/chatEditorActions.ts

```diff
@@ -36,7 +38,7 @@ abstract class NavigateAction extends Action2 {
 					? KeyMod.Alt | KeyCode.F5
 					: KeyMod.Alt | KeyMod.Shift | KeyCode.F5,
 				weight: KeybindingWeight.EditorContrib,
-				when: ContextKeyExpr.and(ctxHasEditorModification, EditorContextKeys.focus),
+				when: ContextKeyExpr.and(ContextKeyExpr.or(ctxHasEditorModification, ctxNotebookHasEditorModification), EditorContextKeys.focus),
```

Comment:
The `ctxHasEditorModifcations` only applies to regular editors & not notebooks, hence the new context var

Reply:
Alternatively I can create new menu items and ensure these are disabled for notebooks.